SwiftLeeds/Views/ContentView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ContentView.swift
+//  SwiftLeeds
+//
+//  Created by Matthew Gallagher on 14/11/2021.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        ZStack {
+            Color.background
```

Comment:
I know it's just an intro page but if you add `.ignoresSafeArea(.all)` to the top level it will mean that text could grow off the page with a huge accessibility font size. If you set it on just the color you get the full-bleed effect you're after without the content being allowed to just grow and grow 😄 

Can re-visit when it comes to making more screens but you can actually do a neat trick where you set the background to fill all available space, and then bound the rest of the content to the readableContentGuide, meaning you can forget about it apart from in one top level wrapper

Reply:
I've moved the ignore to the colour itself so the content will still respect those areas.